src/plays/Faisalplay/Readme.md

```diff
@@ -0,0 +1,25 @@
+# Stopwatch
+
+great play lovely play enjoy play
+
+## Play Demographic
+
+- Language: ts
+- Level: Beginner
+
+## Creator Information
+
+- User: irfanmit
+- Gihub Link: https://github.com/irfanmit
+
+## Implementation Details
+
+Update your implementation idea and details here
+
+## Consideration
+
+Update all considerations(if any)
+
+## Resources
+
+Update external resources(if any)
```

Comment:
please provide the basic details if there are nothing then remove extra things

Reply:
i have added the basic details here